### PR TITLE
Go cambridge

### PR DESCRIPTION
**Pull request by AnnaGoodman1**
ready for checking

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Hi @nikolai-b yes I'm planning to build and tile the rasters Friday next week and then we can deploy the data in one go. No harm in putting the intermediate product (without rasters) on the test server. Did you get a chance to test the app on this branch? Good point regarding the JS front page, hadn't thought about that. Let us know how you update the landing map: it should be reproducible.

File: regions_www/www/front_page/map.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ $(document).ready(function(){
     govtarget_slc_perc: "Government Target (equality)",
     govnearmkt_slc_perc: "Government Target (near market)",
     gendereq_slc_perc: "Gender Equality",
+    cambridge_slc_perc: "Go Cambridge",
```
[the remark this answers is not in the document]
Comment (AnnaGoodman1) [reply regions_www/www/front_page/map.js]: yes please both to front page and to disabling Go Camb in commute layer on front page (I think it is already disabled within map, unless I messed that up)

**Comment by nikolai-b** [comment]
Sure, I'll update the front page.  

> Do we also need the route network, at the moment it looks blank?

@Robinlovelace deploying means it is not only on the test server but it means it is ready for production.  If we wanted to deploy another fix to production then it would mean this change would also be deployed so although it is fine to get things ready I'd prefer to only merge them in till if there are known issues.  Does this have a deadline?

**Comment by Robinlovelace** [comment]
Plan to look at this on Friday. No hard deadline, but the sooner the better.

**Comment by AnnaGoodman1** [comment]
ready for test server once @nikolai-b happy, I think

**Comment by Robinlovelace** [comment]
Merging now to test deploy to test server over the weekend. Plan to push this to production Monday, Tuesday or Weds. Sound good @AnnaGoodman1 ? And did you plan to write a blog post about this? I think that would be a good idea, to explain and and link to the updated documentation if nothing else. Heads-up @RachelAldred @JDWoodcock this is moving forward :rocket:

**Comment by JDWoodcock** [comment]
looking forward to it

**Pull request merged**